packages/gatsby/package.json

```diff
@@ -89,105 +87,97 @@
     "gatsby-react-router-scroll": "^4.0.0-next.1",
     "gatsby-telemetry": "^2.0.0-next.1",
     "glob": "^7.1.6",
-    "got": "8.3.2",
-    "graphql": "^15.4.0",
+    "got": "11.8.1",
+    "graphql": "^15.5.0",
     "graphql-compose": "~7.25.0",
-    "graphql-playground-middleware-express": "^1.7.18",
-    "hasha": "^5.2.0",
+    "graphql-playground-middleware-express": "^1.7.22",
+    "hasha": "^5.2.2",
     "http-proxy": "^1.18.1",
     "invariant": "^2.2.4",
     "is-relative": "^1.0.0",
     "is-relative-url": "^3.0.0",
-    "jest-worker": "^24.9.0",
-    "joi": "^17.2.1",
+    "jest-worker": "^26.6.2",
+    "joi": "^17.4.0",
     "json-loader": "^0.5.7",
     "json-stringify-safe": "^5.0.1",
     "latest-version": "5.1.0",
-    "lodash": "^4.17.20",
+    "lodash": "^4.17.21",
     "md5-file": "^5.0.0",
-    "meant": "^1.0.1",
+    "meant": "^2.0.1",
     "memoizee": "^0.4.15",
     "micromatch": "^4.0.2",
-    "mime": "^2.4.6",
+    "mime": "^2.5.2",
     "mini-css-extract-plugin": "^1.3.8",
-    "mitt": "^1.2.0",
-    "mkdirp": "^0.5.1",
-    "moment": "^2.27.0",
+    "mitt": "^2.1.0",
+    "moment": "^2.29.1",
     "name-all-modules-plugin": "^1.0.1",
     "normalize-path": "^3.0.0",
     "null-loader": "^4.0.1",
-    "opentracing": "^0.14.4",
+    "opentracing": "^0.14.5",
     "p-defer": "^3.0.0",
     "parseurl": "^1.3.3",
     "physical-cpu-count": "^2.0.0",
-    "pnp-webpack-plugin": "^1.6.4",
-    "postcss": "8.2.5",
-    "postcss-flexbugs-fixes": "^4.2.1",
+    "postcss": "8.2.6",
+    "postcss-flexbugs-fixes": "^5.0.2",
     "postcss-loader": "^5.0.0",
-    "prompts": "^2.3.2",
+    "prompts": "^2.4.0",
     "prop-types": "^15.7.2",
-    "query-string": "^6.13.1",
+    "query-string": "^6.14.0",
     "raw-loader": "^4.0.2",
-    "react-dev-utils": "^11.0.1",
-    "react-error-overlay": "^3.0.0",
-    "react-hot-loader": "^4.13.0",
+    "react-dev-utils": "^11.0.3",
     "react-refresh": "^0.9.0",
     "redux": "^4.0.5",
     "redux-thunk": "^2.3.0",
-    "semver": "^7.3.2",
+    "semver": "^7.3.4",
     "shallow-compare": "^1.2.2",
     "signal-exit": "^3.0.3",
-    "slugify": "^1.4.4",
-    "socket.io": "2.3.0",
-    "socket.io-client": "2.3.0",
+    "slugify": "^1.4.7",
+    "socket.io": "3.1.1",
+    "socket.io-client": "3.1.1",
     "source-map": "^0.7.3",
     "source-map-support": "^0.5.19",
     "st": "^2.0.0",
     "stack-trace": "^0.0.10",
-    "string-similarity": "^1.2.2",
-    "strip-ansi": "^5.2.0",
+    "string-similarity": "^4.0.4",
     "style-loader": "^2.0.0",
     "terser-webpack-plugin": "^5.1.1",
     "tmp": "^0.2.1",
     "true-case-path": "^2.2.1",
     "type-of": "^2.0.1",
     "url-loader": "^4.1.1",
-    "util.promisify": "^1.0.1",
-    "uuid": "3.4.0",
+    "util.promisify": "^1.1.1",
+    "uuid": "8.3.2",
```

Comment:
I think we pinned it because the new versions included more stuff we didn't need. I recall some conversation on Slack about this.

Reply:
ok 👍